packages/core/test/support/in-memory-clients.ts

```diff
@@ -0,0 +1,95 @@
+import type * as Oxigraph from 'oxigraph'
```

Comment:
This modules exports im-memory oxigraph wrapped to match the interface of `sparql-http-client`